core/domain/services/assets/EspressoLegacyAdminAssetManager.php

```diff
@@ -46,48 +78,140 @@ public function addAssets()
+            ]
+        )->setEnqueueImmediately();
+
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_EE_DIALOG,
+            EE_ADMIN_URL . 'assets/ee-dialog-helper.js',
+            [
+                JqueryAssetManager::JS_HANDLE_JQUERY,
+                JqueryAssetManager::JS_HANDLE_JQUERY_UI_DRAGGABLE,
+            ]
+        );
+
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_EE_ADMIN,
+            EE_ADMIN_URL . 'assets/ee-admin-page.js',
+            [
+                CoreAssetManager::JS_HANDLE_CORE,
+                EspressoLegacyAdminAssetManager::JS_HANDLE_PARSE_URI,
+                EspressoLegacyAdminAssetManager::JS_HANDLE_EE_DIALOG,
+            ]
+        );
+
+        // script for sorting tables
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_EE_AJAX_TABLE_SORTING,
+            EE_ADMIN_URL . 'assets/espresso_ajax_table_sorting.js',
+            [
+                EspressoLegacyAdminAssetManager::JS_HANDLE_EE_ADMIN,
+                JqueryAssetManager::JS_HANDLE_JQUERY_UI_SORTABLE,
+            ]
+        )->setEnqueueImmediately();
+
+        // script for parsing uri's
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_PARSE_URI,
+            EE_GLOBAL_ASSETS_URL . 'scripts/parseuri.js'
+        );
+
+        // and parsing associative serialized form elements
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_EE_SERIALIZE_FULL_ARRAY,
+            EE_GLOBAL_ASSETS_URL . 'scripts/jquery.serializefullarray.js',
+            [JqueryAssetManager::JS_HANDLE_JQUERY]
+
+        );
+
+        // helpers scripts
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_EE_TEXT_LINKS,
+            EE_PLUGIN_DIR_URL . 'core/helpers/assets/ee_text_list_helper.js',
+            [JqueryAssetManager::JS_HANDLE_JQUERY]
+        );
+
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_MOMENT_CORE,
+            EE_THIRD_PARTY_URL . 'moment/moment-with-locales.min.js'
+        );
+
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_MOMENT,
+            EE_THIRD_PARTY_URL . 'moment/moment-timezone-with-data.min.js',
+            [EspressoLegacyAdminAssetManager::JS_HANDLE_MOMENT_CORE]
+        );
+
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_EE_DATEPICKER,
+            EE_ADMIN_URL . 'assets/ee-datepicker.js',
+            [
+                JqueryAssetManager::JS_HANDLE_JQUERY_UI_TIMEPICKER_ADDON,
+                EspressoLegacyAdminAssetManager::JS_HANDLE_MOMENT,
+            ]
+        );
+
+        // google charts
+        $this->addJavascript(
+            EspressoLegacyAdminAssetManager::JS_HANDLE_GOOGLE_CHARTS,
+            'https://www.gstatic.com/charts/loader.js'
+        );
```

Comment:
A question not related to this PR. Do we add[ Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) for external scripts?

Reply:
oooo no we do not, that would be a nice enhancement though